tests/integ/test_dataframe.py

```diff
@@ -3698,3 +3699,69 @@ def test_dataframe_to_local_iterator_isolation(session):
     assert (
         row_counter == ROW_NUMBER
     ), f"Expect {ROW_NUMBER} rows, Got {row_counter} instead"
+
+
+def test_csv(session, tmpdir_factory):
+    """Tests for df.write.csv()."""
```

Comment:
```suggestion
def test_writer_csv(session, tmpdir_factory):
    """Tests for df.write.csv()."""
```
preferable move this test to `tests/integ/scala/test_dataframe_writer_suite` so it is present with other dataframe writer tests